cpp/ql/lib/semmle/code/cpp/Namespace.qll

```diff
@@ -174,7 +174,26 @@ class UsingDeclarationEntry extends UsingEntry {
    */
   Declaration getDeclaration() { usings(underlyingElement(this), unresolveElement(result), _, _) }
 
-  override string toString() { result = "using " + this.getDeclaration().getDescription() }
+  /**
+   * Returns the member referenced by this using declaration when it refers
+   * to a member of a template type parameter.
```

Comment:
Trying to make it a bit more uniform with the `getDeclaration` QLDoc, and using the word "depends", which is what the standard uses:
```suggestion
   * Gets the member that is referenced by this using declaration, where the member depends on a
   * type template parameter.
```